README.md

```diff
@@ -2,7 +2,7 @@
 
 **Goals**
 
-- Illustrate how to use linkerd in kubernetes
+- Illustrate how to use linkerd and namerd in production-like environments
```

Comment:
I think you should add a link somewhere in this main README.md file to the document you're adding in k8s/README.md.